pages/bibliography.md

```diff
@@ -1,11 +1,16 @@
+{% include nav.html %} {% include css.html %}
+
 # Bibliography
 
 ---
-
-## Project Bibliography
-
-1. List your bibliography items here, whether historical and/or digital in nature, here.
-2. It should be in [Chicago Manual of Style](chicagomanualofstyle.org)
-3. Remember to *italicize* and **bold** as appropriate.
-4. See example below
-5. Clinton, Kevin. *Eleusis, the Inscriptions on Stone: Documents of the Sanctuary of the Two Goddesses and Public Documents of the Deme*. Vol. 1a: Text. Athens: The Archaeological Society at Athens, 2008. 
+1. Bradford, Alfred S. Flying the Black Flag: A Brief History of Piracy. Non-Series. Westport, Conn: Praeger, 2007. http://ezproxy.lib.usf.edu/login?url=http://search.ebscohost.com/login.aspx?direct=true&db=nlebk&AN=218223&site=eds-live. 
+2. Burg, B.R. *Sodomy and the Pirate Tradition: English Sea Rovers in the Seventeenth- Century Caribbean*, (New York: New York University Press, 1984).
+3. Campbell, Gwyn. “Piracy in the Indian Ocean World.” Interventions: The International Journal of Postcolonial Studies 16, no. 6 (November 2014): 775-794
+4. Hoefferle. Caroline, *The Essential Historiography Reader*, (Boston, Pearson Hall, 2011), 172
+5. O’Driscoll, Sally. “The Pirate’s Breasts: Criminal women and the meaning of the body”, The Eighteenth Century 53, No. 3 (2012): 357-379
+6. Rediker, Marcus. ““Under the Banner King Death”: The Social world of Anglo- American Pirates. 1712-1726” The William and Mary Quarterly 38, No.2 (1981): 203-227
+7. Starkey, David J. *Pirates and Privateers: New Perspectives on the War on Trade in the Eighteenth and Nineteenth Centuries*,(Exeter: University of Exeter Press, 1997): 2
+8. Taylor, *Alan American Revolutions: A Continental History, 1750-1804* (New York: W.W. Norton & Company, Inc., 2016) 
+9. https://scholarcommons.usf.edu/cgi/viewcontent.cgi?article=5888&context=etd
+10. https://thesocietypages.org/socimages/2011/01/07/fascinating-database-about-the-trans-atlantic-slave
+11. https://www.slavevoyages.org/voyage/database-trade/ 
```

Comment:
You need to give more information than this. There is full credit information at all of these pages, at least give authors or institutions attached.